src/PTFAnalysis.cpp

```diff
@@ -11,10 +11,11 @@
 #include <fstream>
 #include <math.h>
 
+// pulse charge (integrated pulse height over 2151ns to 2254ns)
```

Comment:
Really naïve question but how do we know, select this interval, specification of the PMT ?

Reply:
Hi! The pulse charge calculation function _ChargeSum()_ was updated in later commits to take in optional arguments to specify the interval (using _bin_low_ and _bin_high_, which corresponds to the time in ns (bin number*8) in the waveform. For the current setup, the PMT laser pulse is observed around 2180-2300ns, so the interval passed in is {bin_low=272,bin_high=287) in channel 2. If the laser pulse time is changed, it will be necessary to manually enter in a new range when calling ChargeSum (~line 727).